packages/dd-trace/test/appsec/sdk/track_event.spec.js

```diff
@@ -129,23 +132,44 @@ describe('track_event', () => {
         })
         expect(prioritySampler.setPriority)
           .to.have.been.calledOnceWithExactly(rootSpan, USER_KEEP, SAMPLING_MECHANISM_APPSEC)
+        expect(sample).to.have.been.calledOnceWithExactly(rootSpan)
+        expect(waf.run).to.have.been.calledOnceWithExactly({
+          persistent: {
+            [LOGIN_SUCCESS]: null,
+            [USER_ID]: 'user_id',
+            [USER_LOGIN]: 'user_id'
+          }
+        })
       })
 
-      it('should call waf run with login success address', () => {
-        const user = { id: 'user_id' }
+      it('should call waf with user login', () => {
+        const user = { id: 'user_id', login: 'user_login' }
 
         trackUserLoginSuccessEvent(tracer, user)
-        sinon.assert.calledOnceWithExactly(
-          waf.run,
-          { persistent: { [LOGIN_SUCCESS]: null } }
-        )
+
+        expect(log.warn).to.not.have.been.called
+        expect(setUserTags).to.have.been.calledOnceWithExactly(user, rootSpan)
+        expect(rootSpan.addTags).to.have.been.calledOnceWithExactly({
+          'appsec.events.users.login.success.track': 'true',
+          '_dd.appsec.events.users.login.success.sdk': 'true'
+        })
+        expect(prioritySampler.setPriority)
+          .to.have.been.calledOnceWithExactly(rootSpan, USER_KEEP, SAMPLING_MECHANISM_APPSEC)
+        expect(sample).to.have.been.calledOnceWithExactly(rootSpan)
+        expect(waf.run).to.have.been.calledOnceWithExactly({
+          persistent: {
+            [LOGIN_SUCCESS]: null,
+            [USER_ID]: 'user_id',
+            [USER_LOGIN]: 'user_login'
+          }
+        })
       })
     })
 
     describe('trackUserLoginFailureEvent', () => {
       it('should log warning when passed invalid userId', () => {
-        trackUserLoginFailureEvent(tracer, null, false)
-        trackUserLoginFailureEvent(tracer, [], false)
+        trackUserLoginFailureEvent(tracer, null, false, { key: 'value' })
```

Comment:
because it's written exactly like this for `trackUserLoginSuccessEvent` tests (example line 69), and it's also optional there anyway. Just wanted some consistency.